Makefile.include

```diff
@@ -10,6 +10,13 @@ RIOTCPU := $(abspath $(RIOTCPU))
 RIOTBOARD ?= $(RIOTBASE)/boards
 RIOTBOARD := $(abspath $(RIOTBOARD))
 
+# Top level git root of the project being built
+RIOTPROJECT ?= $(shell git rev-parse --show-toplevel)
+RIOTPROJECT := $(abspath $(RIOTPROJECT))
```

Comment:
I was just confused because the former name of todays "applications" is "project' .. didn't check the actual value ;)

Anyway: This practically forbids using RIOT sources outside git AND it already exists as `RIOTBASE`.